content/javascript/concepts/dates/terms/now/now.md

```diff
@@ -54,3 +54,16 @@ console.log(monthsSince1970);
 // Output: 627
 // Note: your output may vary
 ```
+
+## Codebyte Example
+
+The following executable example returns the current timestamp in milliseconds since January 1, 1970 00:00:00 UTC:
+
+> **Note**: The code will print different values each time it is executed.
+
+```codebyte/javascript
```

Comment:
In the new example please remove the extra empty lines at the top and bottom of the code block